src/services/types.ts

```diff
@@ -71,6 +71,10 @@ namespace ts {
         getLineAndCharacterOfPosition(pos: number): LineAndCharacter;
     }
 
+    export interface SourceMapSource {
+        /* @internal */ getLineAndCharacterOfPosition(pos: number): LineAndCharacter;

```

Comment:
This was intentional to prevent `getLineAndCharacterOfPosition` from being written to `typescript.d.ts` as part of `SourceMapSource` and making supplying a `getLineAndCharacerOfPosition()` required when creating a `SourceMapRange` for calls to `setSourceMapRange()`.